### PR TITLE
fix module loading on pudb2

### PR DESCRIPTION
**Pull request by albfan**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -49,6 +49,6 @@
           ],
       packages=["pudb"],
       entry_points={
-          'console_scripts': ['pudb' + PY_VERSION + ' = pudb.run:main'],
+          'console_scripts': ['pudb2' + PY_VERSION + ' = pudb.run:main'],
```
Comment (asmeurer) [inline setup.py]: This is wrong. The PY_VERSION is already added to "pudb".

File: setup.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 finally:
     readme.close()
 
-setup(name='pudb',
+setup(name='pudb2',
```
Comment (asmeurer) [inline setup.py]: The package name should be pudb, not pudb2.

**Comment by asmeurer** [comment]
Can you specify what bug this fixes, and how to reproduce it?

File: setup.py (hunk 1 above)
Comment (albfan) [inline setup.py]: Sorry, just want to PR second commit. My fault

**Comment by albfan** [comment]
Sorry this was my very own tag. Have to check tip

**Pull request closed without merging**